Services/OperationService/Commands/GetPortfolioCommand.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System.Collections.Generic;
+using System.Threading.Tasks;
+using Interfaces;
+using DTO;
+using DTO.BrokerRequests;
+using MassTransit;
+using Microsoft.AspNetCore.Mvc;
+using Kernel;
+using Kernel.CustomExceptions;
+
+namespace OperationService.Commands
+{
+    public class GetPortfolioCommand : ICommand<GetPortfolioRequest, List<InstrumentData>>
+    {
+        private IRequestClient<GetPortfolioRequest> client;
+
+        public GetPortfolioCommand([FromServices] IRequestClient<GetPortfolioRequest> client)
+        {
+            this.client = client;
```

Comment:
logs

Reply:
> logs

Ok, but I really don't like logging like EVERY FREAKING THING, generating thousands of logs after using app just for a few hours, for example, and this only includes usage by a single person
🙃